### PR TITLE
Remove sftp from export service

### PR DESCRIPTION
**Pull request by hbrace**
# Motivation and Context

* [ ] [Context Index](/CODE_GUIDE.md#context-index) has been kept up to date

We no longer want to use sftp as the place to put export files, instead for now we only want to use GCS buckets

# What has changed

Everything sftp removed, changes made to make writing to GCS buckets more concrete, configurable to use local file system when running locally.

# How to test?

Running locally
1. Build export file service image
3. with `ssdc-rm-docker-dev` on branch of the same name: `make up`
4.`make test` to run the tests

Running locally using real GCS buckets:
1. Follow the new instructions on the export file service to set variables
2. Do the same in the acceptance tests
3. `make test`

Running in GCP
1. Apply terraform to your env from the branch of the same name
1. `make docker_build`
2. In the export file service repo, `docker tag eu.gcr.io/ons-ci-rm/rm/ssdc-rm-export-file-service:latest eu.gcr.io/ssdc-rm-<<YOUR-ENV>>/rm/ssdc-rm-export-file-service:latest
3. `docker push eu.gcr.io/ssdc-rm-<<YOUR-ENV>>/rm/ssdc-rm-export-file-service`
4. Edit export file service manifest file in `ssdc-rm-kubernetes` to use your pushed image and apply this
5.  Build docker image, tag and push the acceptance tests to your env in the same way
6. In acceptance tests directory, edit the `acceptance_tests_pod.yml` to point to your envs AT image
7. `./run_gke`

# Links

https://trello.com/c/tqkddH1D/3019-export-file-service-remove-sftp-just-send-files-to-a-gcs-bucket-or-local-directory-8
# Screenshots (if appropriate):

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -73,3 +67,8 @@ class Config:
     RESOURCE_FILE_PATH = Path(os.getenv('RESOURCE_FILE_PATH') or Path(__file__).parent.joinpath('resources'))
 
     SAMPLE_LOAD_ORIGINATING_USER = os.getenv('ORIGINATING_USER', 'dummy@fake-email.com')
+
+    SENT_EXPORT_FILE_BUCKET = os.getenv('SENT_EXPORT_FILE_BUCKET',
+                                        '/tmp')
+
+    PLATFORM = os.getenv('PLATFORM', 'LOCAL')
```
Comment (ryangrundy7) [inline config.py:74]: These variables aren't being used anywhere according to vulture?

File: acceptance_tests/features/steps/export_file.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 from behave import step
 from tenacity import retry, retry_if_exception_type, stop_after_delay, wait_fixed
 
-from acceptance_tests.utilities.sftp_helper import SftpUtility
+from acceptance_tests.utilities.export_file_helper import get_all_files_after_time, get_files_content_as_list
```
Comment (ryangrundy7) [inline acceptance_tests/features/steps/export_file.py:11]: Doesn't exist anymore?

File: acceptance_tests_pod.yml [hunk 3]
```diff
@@ -22,7 +22,7 @@ spec:
               cp /home/acceptancetests/postgresql/* /home/acceptancetests/.postgresql
               chmod 0600 /home/acceptancetests/.postgresql/postgresql.key
               # the above is a workaround to get round the permissions as currently we cant mount it without doing this
-    image: eu.gcr.io/ssdc-rm-ci/rm/ssdc-rm-acceptance-tests:latest
+    image: eu.gcr.io/ssdc-rm-braceh1/rm/ssdc-rm-acceptance-tests:latest
```
Comment (ryangrundy7) [inline acceptance_tests_pod.yml:25]: Pointing to your dev env

**Review by ryangrundy7 — CHANGES_REQUESTED** [review]
Looks like the PR is missing some stuff

**Review by ryangrundy7 — APPROVED** [review]
All tests pass 👍

**Review by liamtoozer — APPROVED** [review]
Looks good. Passes locally for me 👍

**Pull request merged**